### PR TITLE
add quarto render gh actions check

### PR DESCRIPTION
**Pull request by sckott**
This is to check that quarto render works - NOT to preview the site - just to check that `quarto render` succeeds

Previews for PRs is waiting on 
- actions being turned on
- netlify being approved

### CODE REVIEW

**Review by seankross — DISMISSED** [review]

File: .github/workflows/quarto-check.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      contents: write
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Quarto
+        uses: quarto-dev/quarto-actions/setup@v2
+
+      - name: Dependencies
+        run: quarto install tinytex
```
Comment (seankross) [inline .github/workflows/quarto-check.yml:23]: Maybe it doesn't matter but is it `quarto install tinytex` or `quarto install tool tinytex`? (See: https://github.com/quarto-dev/quarto-actions/tree/main/setup#:~:text=quarto%20install%20tool%20tinytex)

File: .github/workflows/quarto-check.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+name: Check Quarto Render
+
+jobs:
+  build-deploy:
```
Comment (seankross) [inline .github/workflows/quarto-check.yml:11]: Would you mind changing this to: `check-build`?

**Review by seankross — CHANGES_REQUESTED** [review]
See inline comments

File: .github/workflows/quarto-check.yml (hunk 1 above)
Comment (sckott) [reply .github/workflows/quarto-check.yml:23]: ah looks like it's just a var within setup, ill change

**Review by seankross — APPROVED** [review]

**Pull request merged**